app/services/notification_service.rb

```diff
@@ -1,6 +1,13 @@
 class NotificationService
 
-  def self.notify(account:, project:, issue_id:, issue_comment_id: nil)
+  def self.notify(account_id:, project_id:, issue_id:, issue_comment_id: nil)
+    project = Project.find(project_id)
```

Comment:
Since we only use the id, I don't think we need this database call.

Reply:
we use it [here](https://github.com/ContributorCovenant/beacon/blob/release/app/services/notification_service.rb#L20)

Reply:
I see, okay